schema/bench-params.json

```diff
@@ -26,6 +26,9 @@
             "server",
             "all"
           ]
+        },
+        "id": {
+          "type": "string"
```

Comment:
We should probably impart some requirements here, right?  As in in can't be empty and it should be a numerical value, right?